src/Akeneo/Pim/Enrichment/Component/Product/Connector/Processor/QuickExport/ProductAndProductModelProcessor.php

```diff
@@ -140,6 +140,8 @@ function ($attributeCode) use ($selectedProperties) {
                 );
             } elseif (in_array($codeProperty, $selectedProperties) || 'identifier' === $codeProperty) {
                 $propertiesToExport[$codeProperty] = $property;
+            } elseif ('code' === $codeProperty) {
+                $propertiesToExport['identifier'] = $property;
```

Comment:
For me you just need this, the writer doesn't need to know that something the identifier is in identifier
```suggestion
            } elseif (in_array($codeProperty, $selectedProperties) || 'identifier' === $codeProperty || 'code' === $codeProperty) {
                $propertiesToExport[$codeProperty] = $property;
            }
```

Reply:
If we do that, the code column and value will be present in the exported csv/xlsx file even if it is not in the grid context.